### PR TITLE
(DM-2367) Feature/run_outside_lsst-dev

### PR DESCRIPTION
**Pull request by jhoblitt**

### CODE REVIEW

**Comment by frossie** [comment]
I assume you've checked that renaming BUILDIR doesn't have knock-on effects - but I am happy to tolerate minor breakage for this much cleaning up anyway.

File: bin/deploy [hunk 1]
```diff
@@ -3,21 +3,22 @@
 #  Deploy a standardized, standalone, lsst-build sandbox
 #
 
+SCRIPT_DIR=${0%/*}
+source ${SCRIPT_DIR}/../etc/settings.cfg.sh
+
 EUPS_VERSION=${EUPS_VERSION:-1.5.4}		# Version of EUPS to install
```
Comment (ktlim) [inline bin/deploy]: I know this has been merged, but isn't eups 1.5.7 current?

File: bin/deploy [hunk 2]
```diff
@@ -103,19 +104,34 @@ test -f "$LSSTSW/stack/.deployed" || ( # Adjust the stack config
 test -f "$LSSTSW/versiondb/.deployed" || ( # Clone the version database
     echo "::: Deploying versiondb"
     rm -rf versiondb
-    git clone git@github.com:lsst/versiondb.git
-    (cd versiondb && git config push.default current)
+    if [ -z ${NO_PUSH+x} ]; then
```
Comment (ktlim) [inline bin/deploy]: Isn't this officially spelled `${NO_PUSH:+x}`?  And isn't this backwards?  If NO_PUSH is defined, then the ssh clone and git config are performed.

File: bin/deploy [hunk 3]
```diff
@@ -103,19 +104,34 @@ test -f "$LSSTSW/stack/.deployed" || ( # Adjust the stack config
...
-    git clone git@github.com:lsst/lsst_build.git
+    git clone https://github.com/lsst/lsst_build.git
     (cd lsst_build && git config push.default current)
     touch "$LSSTSW/lsst_build/.deployed"
 )
 
+test -f "${LSSTSW}/lfs/bin/numdiff" || (
+    echo "::: Deploying numdiff"
+    cd ${LSSTSW}/sources
+    wget http://download-mirror.savannah.gnu.org/releases//numdiff/numdiff-5.6.1.tar.gz
```
Comment (ktlim) [inline bin/deploy]: Extra slash.

**Pull request merged**